### DISCUSSION TITLE
What do you think about adding options to skip `archived` and `fork` respectively?

### DISCUSSION THREAD

**Discussion by oscard0m** [octoherd/octoherd#26] [Ideas]
It would be nice to give the option to the users, specially when using `<org>/*` pattern, to skip archived and/or forked repos. What do you think?

I find this pattern in more than one octoherd script:

https://github.com/octoherd/script-remove-dependabot/blob/d4897ff3194c07d3cabb26088673dc7237b1445c/script.js#L9-L12

Should this be a plugin? An cli option?

**Comment by gr2m** [comment]
I thought about it, but I didn't come up with a good solution yet. I think for now we should add a question to https://github.com/octoherd/create-octoherd-script/ asking if the repository should be applied to archived repositories or not. And if the answer is no, we should add the code snipped you linked above.

If we made it a CLI option then people would still be able to run the script without setting the option and it would blow up.

An alternative would be to export configuration for the script which would not be configurable, so besides exporting the `script` function we could support something like `export const ignoreArchived = true` or similar. But that's not something I want to explore yet
